pkg/services/oauth/oauth.go

```diff
@@ -47,16 +67,16 @@ func (s *OAuthService) Configure() (err error) {
 }
 
 // CreateJwtBearerToken creates a JwtBearerTokenResult containing a jwtBearerToken from a CreateJwtBearerTokenRequest.
-func (s *OAuthService) CreateJwtBearerToken(request *services.CreateJwtBearerTokenRequest) (*services.JwtBearerTokenResult, error) {
+func (s *OAuthService) CreateJwtBearerToken(request *services.CreateJwtBearerTokenRequest, endpointIdentifier string) (*services.JwtBearerTokenResult, error) {
 	jti, err := uuid.NewRandom()
 	if err != nil {
 		return nil, err
 	}
 
 	jwtBearerToken := services.NutsJwtBearerToken{
 		StandardClaims: jwt.StandardClaims{
-			//Audience:  endpoint.Identifier.String(),
-			ExpiresAt: time.Now().Add(30 * time.Minute).Unix(),
+			Audience:  endpointIdentifier,
+			ExpiresAt: time.Now().Add(5 * time.Second).Unix(),
```

Comment:
5 seconds leaves very little time for out-of-sync/skewed clocks?

Reply:
Well security related things shouldn't be suggestions....

The 5 sec is based upon counting for some delay. If it can't do a oauth flow within 5 seconds then the client will have to try again.